pkg/event/dispatcher.go

```diff
@@ -46,8 +46,8 @@ type HTTPEventDispatcher struct {
 // DispatchEvent dispatches event with callback
 func (*HTTPEventDispatcher) DispatchEvent(event LogEvent) (bool, error) {
 
-	requester := utils.NewHTTPRequester(event.EndPoint)
-	_, _, code, err := requester.Post(event.Event)
+	requester := utils.NewHTTPRequester()
```

Comment:
Are you planning on tackling the requester re-use in a different PR?

Reply:
yes